### PR TITLE
doc and chore: OLAS bridging

### PR DESCRIPTION
**Pull request by kupermind**
- Adding OLAS bridging information and deployment procedure / scripts;
- Shuffling deployment scripts around to unify Optimism and Base approaches.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,14 +155,46 @@ For running a test between `goerli` and `chiado`, run the test script with your
 and [`goerli-chiado` governor bridge test](https://github.com/valory-xyz/autonolas-governance/blob/main/scripts/deployment/bridges/polygon/test/mediator_goerli_chiado_governor.js).
 Note that the script must be run without Hardhat environment, i.e.: `node test_script.js`.
 
-### ERC20 token bridging between Polygon and Ethereum
-The contract design facilitating token bridging between the Polygon and Ethereum networks, along with the underlying motivations driving the creation of these contracts, is outlined here:
-[Bridging token](https://github.com/valory-xyz/autonolas-governance/blob/main/docs/Bonding_mechanism_with_Polygon_LP_token.pdf?raw=true). 
+#### Arbitrum governance bridge
```
Comment (kupermind) [inline README.md:158]: Adding all available bridging info

File: README.md [hunk 2]
```diff
@@ -155,14 +155,46 @@ For running a test between `goerli` and `chiado`, run the test script with your
...
+- [bridges-gnosis](https://github.com/valory-xyz/autonolas-governance/blob/main/scripts/deployment/bridges/gnosis);
+- [bridges-optimism-base](https://github.com/valory-xyz/autonolas-governance/blob/main/scripts/deployment/bridges/optimistic);
+- [bridges-wormhole](https://github.com/valory-xyz/autonolas-governance/blob/main/scripts/deployment/bridges/wormhole).
+
+### ERC20 token bridging
+Autonolas will use native bridges for ERC20 token transfers, where possible. If a native bridge is not available or
+does not correspond to all the required specifications, the [Wormhole Portal](https://portalbridge.com/advanced-tools/#/transfer)
+is utilized in order to manage the ERC20 token transfers between L1 and L2-s.
+
+For more information about OLAS bridging see [here](https://github.com/valory-xyz/autonolas-governance/blob/main/docs/olas_bridging.md)
```
Comment (kupermind) [inline README.md:192]: OLAS bridging

File: docs/configuration.json [hunk 3]
```diff
@@ -198,5 +198,27 @@
...
+    "contracts": [
+      {
+        "name": "WormholeMessenger",
+        "artifact": "abis/0.8.23/WormholeMessenger.json",
+        "address": ""
+      }
+    ]
+  },
+  {
+    "name": "celoAlfajores",
```
Comment (kupermind) [inline docs/configuration.json:214]: Celo deployed testnet wormhole messenger

File: scripts/audit_chains/audit_contracts_setup.js [hunk 4]
```diff
@@ -364,11 +364,33 @@ async function checkOptimismMessenger(chainId, provider, globalsInstance, config
...
     customExpect(foreignGovernor, globalsInstance["timelockAddress"], log + ", function: foreignGovernor()");
 
-    // Check AMBContractProxyHomeAddress
+    // Check L2CrossDomainMessengerAddress
     const proxyHome = await optimismMessenger.CDMContractProxyHome();
     customExpect(proxyHome, globalsInstance["L2CrossDomainMessengerAddress"], log + ", function: CDMContractProxyHome()");
 }
 
+// Check WormholeMessenger: chain Id, provider, parsed globals, configuration contracts, contract name
+async function checkWormholeMessenger(chainId, provider, globalsInstance, configContracts, contractName, log) {
```
Comment (kupermind) [inline scripts/audit_chains/audit_contracts_setup.js:373]: Adding a check for a wormhole messenger static audit

File: scripts/deployment/bridges/base/README.md [hunk 5]
```diff
@@ -1,47 +0,0 @@
-# Bridge-related deployment scripts
```
Comment (kupermind) [inline scripts/deployment/bridges/base/README.md:1]: Merging Optimism and Base bridging, just leaving out their different globals and tests.

**Review by 77ph — APPROVED** [review]

**Review by 77ph — APPROVED** [review]

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**